terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars

```diff
@@ -0,0 +1,15 @@
+environment = "cidev"
+aws_profile = "development-eu-west-2"
+
+# service configs
+use_set_environment_files = true
+log_level = "trace"
+
+required_cpus = 512
+required_memory = 1024
```

Comment:
Can we remove the `required_memory`? This value can be set as a default value in the variables.tf
`required_memory` as 1024